src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -5,49 +5,56 @@
     "phone" : "94351253",
     "email" : "alice@example.com",
     "address" : "123, Jurong West Ave 6, #08-111",
+    "tags" : [ "friends" ],
     "remark" : "",
-    "tags" : [ "friends" ]
+    "moneyOwed" : "0"
   }, {
     "name" : "Benson Meier",
     "phone" : "98765432",
     "email" : "johnd@example.com",
     "address" : "311, Clementi Ave 2, #02-25",
+    "tags" : [ "owesMoney", "friends" ],
     "remark" : "",
-    "tags" : [ "owesMoney", "friends" ]
+    "moneyOwed" : "0"
```

Comment:
Wait I know the tag is not you do one but one of the tag is owesMoney, maybe you can actually put some value here? You may also want to set some random values for 1 - 2 other people here too.

Reply:
Made changes according to typicalPersons